<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin::Internal::MainWindow</class>
 <widget class="QMainWindow" name="MainWin::Internal::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="patientNamesLayout">
      <item>
       <widget class="QLabel" name="surnamesLabel">
        <property name="text">
         <string>Patient name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="patientName">
        <property name="toolTip">
         <string>Family or occidental last names</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="firstNamesLabel">
        <property name="text">
         <string>Firstname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="patientFirstname">
        <property name="toolTip">
         <string>Given names</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="morePatientInfoButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="DrugsWidget::DrugsCentralWidget" name="m_CentralWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="clearPatient">
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QGroupBox" name="patientInformation">
      <property name="title">
       <string>Patient information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="dobLabel">
           <property name="text">
            <string>DOB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dobDateEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string comment="Format de date">yyyy/MM/dd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Weight</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="patientWeight">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="weightUnit">
           <item>
            <property name="text">
             <string notr="true">kg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">lbs</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="patientSize">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sizeUnit">
           <item>
            <property name="text">
             <string notr="true">cm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ins</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Gender</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sexCombo"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Creatinine clearance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="patientClCr">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="crClUnit">
           <item>
            <property name="text">
             <string notr="true">ml/min</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ml/sec</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Creatinine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="patientCreatinin">
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="creatinineUnit">
           <item>
            <property name="text">
             <string notr="true">mg/l</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">mg/dl</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">mmol/l</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Âµmol/l</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Drugs precautions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="drugsPrecautions">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrugsWidget::DrugsCentralWidget</class>
   <extends>QWidget</extends>
   <header location="global">drugsplugin/drugswidget/drugscentralwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>morePatientInfoButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>patientInformation</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>793</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
